docs/release-notes.txt

```diff
@@ -6,6 +6,49 @@ Release Notes
 Version 0.12
 ------------
 
+Version 0.12.3
+^^^^^^^^^^^^^^
+
+**Release Date:** April 27, 2020
+
+- Add a tutorial for the new (experimental) Native API.
+
+- Add support for locally testing experiments via ``det e create --local``.
+
+- Add :class:`determined.experimental.Determined` class for accessing Experiment, Trial, and Checkpoint objects.
+
+- TensorBoard logs now appear under the ``storage_path`` for ``shared_fs`` checkpoint configurations.
+
+- Allow commands, notebooks, shells, and TensorBoards to be killed before they are scheduled.
+
+- Print container exit reason in trial logs.
+
+- Choose a better default for the ``--tail`` option of command logs.
+
+- Add additional REST API endpoints for trials.
```

Comment:
`additional` feels redundant?
How about `Add REST API endpoints for trials`?

Reply:
done